### PR TITLE
update tabuline and statusline types for v2.5

### PR DESCRIPTION
**Pull request by KorigamiK**

### CODE REVIEW

**Comment by KorigamiK** [comment]
Fixes #254

**Comment by siduck** [comment]
@KorigamiK can you remove the mappings types related to v2.0?

File: nvchad_types/chadrc.lua [hunk 1]
```diff
@@ -85,10 +85,18 @@
...
+---@field order? string[]
+--- Your modules to be added to the statusline
+--- ```lua
+--- modules = {
+---   abc = function()
+---       return "hi"
+---     end,
+--- }
+--- ```
+---@field moules? table<string, fun(): string>
```
Comment (lucario387) [inline nvchad_types/chadrc.lua]: Typos and offputting indents

File: nvchad_types/chadrc.lua [hunk 2]
```diff
@@ -99,15 +107,22 @@
...
+---@field order? ('"available_space"'|'"treeOffset"'|'"buffers"'|'"tabs"'|'"btns"')[] | string[]
 --- Show numbers on tabufline buffer tabs
---- @field show_numbers? boolean
+---@field show_numbers? boolean
+--- Your modules to be added to the tabufline
+--- ```lua
+--- modules = {
+---   abc = function()
+---       return "hi"
+---     end,
```
Comment (lucario387) [inline nvchad_types/chadrc.lua]: Reindent these

File: nvchad_types/chadrc.lua (hunk 1 above)
Comment (KorigamiK) [reply nvchad_types/chadrc.lua]: got it, thanks :+1:

**Comment by KorigamiK** [comment]
> @KorigamiK can you remove the mappings types related to v2.0?

I believe, these are all I had to remove?

File: nvchad_types/chadrc.lua (hunk 2 above)
Comment (KorigamiK) [reply nvchad_types/chadrc.lua]: done! :smile:

**Review by lucario387 — APPROVED** [review]

**Pull request merged**